src/libs/thirdparty/pikevm/re.c

```
@@ -0,0 +1,803 @@
+        PC += size;
+      }
+      if (maxcnt < 0) {
+        EMIT(PC, rsplit);
+        EMIT(PC+1, REL(PC, PC - size));
+        PC += 2;
+      }
+      else if (maxcnt > 0) {
+        int diff = mincnt == 0 ? 1 : 0;
+        for (int i = maxcnt - mincnt - diff; i > 0; i--) {
+          EMIT(PC++, split);
+          EMIT(PC++, REL(PC, PC+((size+2)*i)));
```

cppcheck

[error] Expression 'code[prog->unilen++]=(prog->unilen+((size+2)*i)-(prog->unilen)-2)' depends on order of evaluation of side effects

(unknownEvaluationOrder)